src/DataSource/Interpreter/AbstractInterpreter.php

```diff
@@ -212,10 +213,12 @@ protected function processImportRow(array $data)
             $createQueueItem = $this->deltaChecker->hasChanged($this->configName, $this->idDataIndex, $data);
         }
 
+        $userOwner = Tool\Admin::getCurrentUser()?->getId() ?? 0;
```

Comment:
```suggestion
        $userOwner = Admin::getCurrentUser()?->getId() ?? 0;
```

Reply:
Could you please add a comment saying that if there is no user logged in we use the system user as `userOwner`